python/smashbox/utilities/__init__.py

```diff
@@ -657,3 +686,18 @@ def expect_does_not_exist(fn):
     """ Checks that a file does not exist, as expected
     """
     error_check(not os.path.exists(fn), "File %s exists but should not" % fn)
+
+
+def download_file(url, filename):
+    """Downloads a file in a path given (filename) if just the name of the file is given 
+       it is download locally
+    """
+    with open(filename, 'wb') as handle:
+        response = requests.get(url, stream=False)
```

Comment:
Is the "stream=False" ok? I think it should be "stream=True" to download the file as stream. The idea is to avoid loading the whole file in the memory.